decentraland/unity-renderer
# Chore: Optimize UI shaders

## What does this PR change?



...

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=chore/Optimize-UI-Shaders
2. Open the chat and the main menu and check the loading spinners and rainbows is being rendered properly and animated.
4. Open the emotes wheel and check the rainbow on the selection is being render properly and animated.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
